packages/office-ui-fabric-react/src/components/Calendar/CalendarMonth.tsx

```diff
@@ -101,7 +101,8 @@ export class CalendarMonth extends BaseComponent<ICalendarMonthProps, {}> {
                   css('ms-DatePicker-monthOption',
                     styles.monthOption,
                     {
-                      ['ms-DatePicker-day--today ' + styles.monthIsCurrentMonth]: highlightCurrentMonth && this.isCurrentMonth(index, navigatedDate.getFullYear(), today!)
+                      ['ms-DatePicker-day--today ' + styles.monthIsCurrentMonth]: highlightCurrentMonth && this.isCurrentMonth(index, navigatedDate.getFullYear(), today!),
+                      ['ms-DatePicker-day--highlighted ' + styles.monthIsHighlighted]: highlightCurrentMonth && (navigatedDate.getMonth() === index)
```

Comment:
![1](https://user-images.githubusercontent.com/25463697/29685945-e79b2ede-88cb-11e7-916b-ed04480ac6f2.jpeg)
When the day and month picker are inline with each other there is a currentMonth and a highlighted month if the select a different month.